pkg/shell/shell.scss

```diff
@@ -127,10 +127,46 @@ html.index-page body {
         max-height: 20rem;
         overflow: auto;
     }
-}
 
-#display-language-list {
-    margin-bottom: var(--pf-global--spacer--sm);
+    .pf-c-menu {
+        // The widget needs a little spacing between the info text and itself
+        margin-top: var(--pf-global--spacer--sm);
+    }
+
+    // PF overrides to fake a multiselect widget (as one does not currently exist in PF4).
+    // A menu gives us the interaction we want, but the styling is a bit off.
+    // Therefore, we're changing the visuals here locally.
+    // PF4 upstream request for multi-select @ https://github.com/patternfly/patternfly/issues/4027
+    .pf-c-menu {
+        // It's  not really a menu, so it shouldn't have a shadow
+        box-shadow: none;
+
+        // Divider is silly between the widgets in this context
+        .pf-c-divider {
+            display: none;
+        }
+
+        &__content {
+            // An overflow multi-select widget needs an outline
+            border: 1px solid var(--pf-global--BorderColor--100);
+            // There should be minimal space between the widgets (replacing the divider)
+            margin-top: var(--pf-global--spacer--sm);
+        }
+    
```

Comment:
There is a tab here which github does not show. If you repost pls remove No need to repost only for this.

Reply:
Whoops. I'll fix.